fe/src/component/UI/Card.js

```diff
@@ -85,33 +87,44 @@ const CardWrapper = styled.div`
 `;
 
 const Card = (props) => {
+  const ctx = useContext(ModalContext);
```

Comment:
변수명의 줄임말은 나중에 보면 알아보기 어려운 경우가 많아서 쓴다면 프로젝트만의 규칙을 명확하게 정해두면 좋을 것 같습니다! 
여기서 쓰는 경우는 객체에서 destructuring해서 바로 써도 괜찮을것 같아 보이네요~

```suggestion
  const { setClickedId, setModalIsDisplayed } = useContext(ModalContext);
```